primedev/client/audio.cpp

```diff
@@ -49,7 +54,7 @@ EventOverrideData::EventOverrideData(const std::string& data, const fs::path& pa
 	}
 
 	rapidjson_document dataJson;
-	dataJson.Parse<rapidjson::ParseFlag::kParseCommentsFlag | rapidjson::ParseFlag::kParseTrailingCommasFlag>(data);
+	dataJson.Parse<(rapidjson::ParseFlag::kParseCommentsFlag | rapidjson::ParseFlag::kParseTrailingCommasFlag)>(data);
```

Comment:
Extra brackets added. Is there a reason to add them? Genuinely curious ^^

Reply:
At the time the compiler threw an error about this, but that could have been a premature error due to a faulty type definition somewhere.
Will test if this can be reverted.

Reply:
turns out, this was caused by `RAPIDJSON_HAS_STDSTRING` being unset
will revert